detox/android/detox/src/full/java/com/wix/detox/reactnative/ReactNativeIdlingResources.kt

```diff
@@ -108,28 +97,39 @@ class ReactNativeIdlingResources constructor(
             networkIdlingResource?.resume()
         }
     }
+
     fun pauseRNTimersIdlingResource() = timersIdlingResource?.pause()
     fun resumeRNTimersIdlingResource() = timersIdlingResource?.resume()
     fun pauseUIIdlingResource() = uiModuleIdlingResource?.pause()
     fun resumeUIIdlingResource() = uiModuleIdlingResource?.resume()
-    fun pauseJSBridgeIdlingResource() = rnBridgeIdlingResource?.pause()
-    fun resumeJSBridgeIdlingResource() = rnBridgeIdlingResource?.resume()
+
+    fun setBlacklistUrls(urlList: String) {
+        val urlArray = toFormattedUrlArray(urlList)
+        NetworkIdlingResource.setURLBlacklist(urlArray)
+    }
 
     private fun setupMQThreadsInterrogators() {
         if (IdlingRegistry.getInstance().loopers.isEmpty()) {
             val mqThreadsReflector = MQThreadsReflector(reactContext)
-//            val mqUIBackground = mqThreadsReflector.getQueue(FIELD_UI_BG_MSG_QUEUE)?.getLooper() TODO
             val mqJS = mqThreadsReflector.getQueue(FIELD_JS_MSG_QUEUE)?.getLooper()
-            val mqNativeModules = mqThreadsReflector.getQueue(FIELD_NATIVE_MODULES_MSG_QUEUE)?.getLooper()
+            val mqNativeModules =
+                mqThreadsReflector.getQueue(FIELD_NATIVE_MODULES_MSG_QUEUE)?.getLooper()
 
             IdlingRegistry.getInstance().apply {
-//                registerLooperAsIdlingResource(mqUIBackground)
                 registerLooperAsIdlingResource(mqJS)
                 registerLooperAsIdlingResource(mqNativeModules)
             }
         }
     }
 
+    private fun setupUrlBlacklist() {
+        val launchArgs = LaunchArgs()
+        if (launchArgs.hasURLBlacklist()) {
+            val blacklistUrls = launchArgs.urlBlacklist
+            setBlacklistUrls(blacklistUrls)
```

Comment:
Something feel off with the private method calling a public one. My suggestion, push `setBlackUrls` to something private and call it here. As for the publish method, [this](https://github.com/wix/Detox/blob/e2f62b8ad66df1b27ad658f3201d54e08201832f/detox/android/detox/src/full/java/com/wix/detox/reactnative/ReactNativeExtension.kt#L81) is an example of an equivalent case.